express/properties/material.py

```diff
@@ -56,15 +56,18 @@ def unitCellFormula(self):
     @property
     def derived_properties(self):
         derived_properties = []
-        try:
```

Comment:
do we want to keep this try/except block?

Reply:
Let's keep it, but log the error in except?

Reply:
Still not done it seems - try/except is gone

Reply:
I will add it back in, my fault for missing that it had been deleted.